styled/checkout.js

```diff
@@ -15,6 +15,30 @@ export const TopBar = styled.View`
   align-items: center;
 `;
 
+// Alphabetical Tab Bottom Bar
+// TODO: flex value fixes height of alphabetical bar to match height of checkout button. Is there a better way to do this?
+
+export const BottomBar = styled.View`
+  height: 57px;
+  width: 100%;
+  background: ${Colors.base};
+  flex: 0.44;
```

Comment:
I'm curious what does this do? When I take it out nothing seems to be affected.

Reply:
omg wat it works without -- amazing
thank u for catching this, it was making me nervous :' )